NeuroID/src/main/java/com/neuroid/tracker/utils/Constants.kt

```diff
@@ -9,5 +9,7 @@ enum class Constants(val displayName: String) {
     debugEventTag("Event"),
     fpjsProdDomain("https://advanced.neuro-id.com"),
     productionEndpoint("https://receiver.neuroid.cloud/"),
+    productionScriptsEndpoint("http://scripts.neuro-id.com/"),
     devEndpoint("https://receiver.neuro-dev.com/"),
+    devScriptsEndpoint("http://scripts.neuro-dev.com/")
```

Comment:
do we use this anywhere or its just in for dev work/testing?

Reply:
Yes, lets make sure that we are using the dev endpoint for testing since an PagerDuty alert is triggered if the test config is not present in prod and throws 404's.

Reply:
only used for dev testing. we should consider leaving this dev for now so we don't set off pager duty until it is ready.